src/components/EarningAreaChart/TooltipContent.tsx

```diff
@@ -0,0 +1,220 @@
+      hasOthers = true
+    }
+
+    return {
+      visibleTokens,
+      hasOthers,
+    }
+  }, [tokens])
+
+  return (
+    <TokensWrapper>
+      {visibleTokens.map((token, i) => {
+        return (
+          <Flex
+            key={i}
+            alignItems="center"
+            sx={{
+              gap: '4px',
+            }}
+          >
+            <Flex
+              sx={{
+                position: 'relative',
+                flex: '0 0 14px',
+                height: '14px',
+                alignItems: 'center',
+                justifyContent: 'center',
+              }}
+            >
+              <Logo srcs={[token.logoUrl]} style={{ width: 14, height: 14 }} />
+              <NetworkLogo
+                chainId={token.chainId}
+                style={{
+                  position: 'absolute',
+                  top: '-4px',
+                  right: '-4px',
+                  width: '10px',
+                  height: '10px',
+                }}
+              />
+            </Flex>
+            <Text
+              as="span"
+              sx={{
+                fontWeight: 400,
+                fontSize: '12px',
+                lineHeight: '14px',
+              }}
+            >
+              {formatTokenAmount(token.amount)}
+            </Text>
+          </Flex>
+        )
+      })}
+
+      {hasOthers && (
+        <Flex
+          alignItems="center"
+          sx={{
+            gap: '4px',
+          }}
+        >
+          <Flex sx={{ width: 16, height: 16, borderRadius: '999px', bg: theme.subText }} />
+          <Text
+            as="span"
+            sx={{
+              fontWeight: 400,
+              fontSize: '12px',
+              lineHeight: '14px',
+            }}
+          >
+            <Trans>Others</Trans>
+          </Text>
+        </Flex>
+      )}
+    </TokensWrapper>
+  )
+}
+
+type Props = {
+  setHoverValue: React.Dispatch<React.SetStateAction<number | null>>
+  dataEntry: EarningStatsTick
+}
+const TooltipContent: React.FC<Props> = ({ dataEntry, setHoverValue }) => {
+  const theme = useTheme()
+
+  useEffect(() => {
+    setHoverValue(dataEntry.totalValue)
+  }, [dataEntry.totalValue, setHoverValue])
+
+  return (
+    <Flex
+      padding="12px"
+      width="min-content"
+      flexDirection="column"
+      sx={{
+        gap: '8px',
+        background: rgba(theme.buttonBlack, 0.9),
+        borderRadius: '12px',
+        border: `1px solid ${darken(0.2, theme.border)}`,
+      }}
+    >
+      <Text
+        as="span"
+        sx={{
+          fontSize: '10px',
+          color: theme.subText,
+        }}
+      >
+        {dataEntry.date}
+      </Text>
+
+      <Text
+        as="span"
+        sx={{
+          fontWeight: 500,
+          fontSize: '14px',
+          lineHeight: '16px',
+          color: theme.text,
+          whiteSpace: 'nowrap',
+        }}
+      >
+        <Trans>My Total Earnings</Trans>: <span>{formatUSDValue(dataEntry.totalValue)}</span>
```

Comment:
why need additional span
```suggestion
        <Trans>My Total Earnings</Trans>: {formatUSDValue(dataEntry.totalValue)}
```